{{cookiecutter.project_slug}}/Dockerfile_template

```diff
@@ -0,0 +1,12 @@
+FROM python:{PYTHON_VERSION}-slim
+
+ENV PYTHONUNBUFFERED=1
+ENV PATH=$PATH:/root/.local/bin
+WORKDIR /project
+
+COPY . /project/
+
+RUN python3 -m pip install --upgrade pip && \
+    python3 -m pip install --user pipx --no-warn-script-location && \
+    python3 -m pipx ensurepath && \
+    python3 -m pip install -r requirements.txt
```

Comment:
It seems we do not have a requirements.txt by default. We might need to generate it before running this line

Reply:
Yes, I'll add an invoke task to handle this part.